home.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Week first Assigment</title>
+    <link rel="stylesheet" href="home.css">
+</head>
+
+<body>
+
+    <div class="header">
+        <nav class="nav" >
+            <img class="imge" src="Images/i.png">
+            <ul class="container">
+                <li>
+                    <a href="#"> Home </a>
+                </li>
+                <li>
+                    <a href="#"> Service </a>
+                </li>
+                <li>
+                    <a href="#"> About </a>
+                </li>
+                <li> <a href="#"> Contact </a>
+                </li>
+                <li>
+                    <a href="#"> Blog </a>
+                </li>
+            </ul>
+
+            <button class="log">Log IN</button>
+
+        </nav>
+
+
+        <div class="conatiner1">
```

Comment:
Please don't add 1 or 2 in naming. Instead please specify proper names. Like "main-container", "body-container" etc.